### ISSUE TITLE
credential_lists in CreateRegistrationResponse

### ISSUE THREAD

**Issue by stef** [cfrg/draft-irtf-cfrg-opaque#87]
in this section https://github.com/cfrg/draft-irtf-cfrg-opaque/blob/master/draft-irtf-cfrg-opaque.md#registration-messages

```
struct {
    opaque data<0..2^16-1>;
    opaque pkS<0..2^16-1>;
    CredentialType secret_types<1..255>;
    CredentialType cleartext_types<0..255>;
} RegistrationResponse;
```
Specifies

> - secret_credentials_list, a list of CredentialType values clients should include in the secret_credentials list of their Credentials structure
> - cleartext_credentials_list, a list of CredentialType values clients should include in the cleartext_credentials list of their Credentials structure

what if a CredentialType is in both of these lists? Why not pack this information in 10 bits, where each CredentialType has 2 bits for signalling:
```c
  NotPackaged = 0,  // field is not packaged in envelope
  InSecEnv = 1,        // field is encrypted
  InClrEnv = 2          // field is plaintext, but authenticated
```

**Comment by stef** [comment]
depending on #84 being removed or reinstated, its also interesting what happens if `skU` is included in clearttext_types. especially if #84 removes the commented out section it also means that the whole information with 2bits per CredentialType would even fit into only one byte, assuming that `skU` is mandatory to be included in secret_types

**Comment by kevinlewi** [comment]
No longer relevant (and so this issue can be closed) since credentials list is not a thing anymore (after #102)
